app/src/main/res/layout/share_editor.xml

```diff
@@ -0,0 +1,18 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="16dp"
+    android:paddingRight="16dp"
+    android:orientation="vertical"
+    android:gravity="start">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/share_editor"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
+    <EditText
+        android:layout_width="match_parent"
```

Comment:
There is a bunch of warnings here that are easily removable according to Cirrus. It might be nice to take it into account so we don't have to refactor it next week ^^.